src/platforms/dotnet/guides/google-cloud-functions/config.yml

```
@@ -0,0 +1,2 @@
+title: Google Cloud Functions
+sdk: sentry.google.cloud.functions
```

Below it's called `sentry.google.cloud.function` (note no **s**` at the end)
Which one is the correct?

```suggestion
sdk: sentry.google.cloud.functions
```

---

I'm using sentry.google.cloud.functions since it's the NuGet package name and sentry.dotnet.google-cloud-function only for the packages.version since it was defined with that name on sentry-release-registry.

---

I believe the name here should match the `sdk.name` so should be `sentry.google.cloud.function`
Or we can bite the bullet and fix the release registry and SDK to report `sentry.google.cloud.functions`